src/lib/proposalHelpers.ts

```diff
@@ -22,7 +22,7 @@ export const closingTime = (proposal: IProposalState) => {
   switch (proposal.stage) {
     case IProposalStage.ExpiredInQueue:
     case IProposalStage.Queued:
-      return moment((proposal.createdAt + proposal.genesisProtocolParams.queuedVotePeriodLimit) * 1000);
+      return moment((Number(proposal.createdAt.toString()) + proposal.genesisProtocolParams.queuedVotePeriodLimit) * 1000);
```

Comment:
Why is `Number(proposal.createdAt.toString())` needed?  Something is awkward there.

Reply:
It is because createdAt can be either a number of Date object: https://github.com/daostack/arc.js/blob/ad9d4c49dd4d25170a6df7e37d5c084f81cb57aa/src/plugins/proposal.ts#L118

Please see this issue for more details: https://github.com/daostack/arc.js/issues/456

Reply:
Can you please put that information in the code?